### ISSUE TITLE
Wrong CLTools version is installed on macOS 10.13 High Sierra.

### ISSUE THREAD

**Issue by JulianArevalo** [elliotweiser/ansible-osx-command-line-tools#33]
Running this role on a macOS 10.13 High Sierra installs old command line tools version.

## Context

Running this role on a macOS 10.13.4 Vagrant box installs correctly the command line tools but not the latest version. The version installed is the one for Xcode 8.2.

Ansible version is 2.5.0

## Process

Run this role with a simple playbook on macOS 10.13:
```yaml
---

- hosts: all
  roles:
    - { role: elliotweiser.osx-command-line-tools }
```
## Expected result

Latest version should be installed.

## Current result
Latest version is not installed, CLTools for Xcode 8.2 are installed instead.
```shell
vag-macos13:~ vagrant$ pkgutil --pkgs | grep CLTools
com.apple.pkg.CLTools_Executables
com.apple.pkg.CLTools_SDK_OSX1012

vag-macos13:~ vagrant$ softwareupdate --history
Display Name                                       Version    Date
------------                                       -------    ----
Command Line Tools (macOS El Capitan version 10.11) for Xcode 8.2        09/05/2018 at 17:21:32

vag-macos13:~ vagrant$ softwareupdate -l
Software Update Tool

Finding available software
Software Update found the following new or updated software:
   * Command Line Tools (macOS High Sierra version 10.13) for Xcode-9.3
        Command Line Tools (macOS High Sierra version 10.13) for Xcode (9.3), 187312K [recommended]
```

**Comment by JulianArevalo** [comment]
The change from #32 fixes the issue, can this PR be merged?

**Comment by elliotweiser** [comment]
@JulianArevalo I'm on it.

**Comment by elliotweiser** [comment]
Fix released in 2.1.0
